Tests/fontBuilder/fontBuilder_test.py

```diff
@@ -81,16 +84,21 @@ def test_build_ttf(tmpdir):
     fb.setupHorizontalHeader(ascent=824, descent=200)
     fb.setupNameTable(nameStrings)
     fb.setupOS2()
-    fb.setupPost()
+    fb.setupPost(keepGlyphNames=keep_glyph_names)
     fb.setupDummyDSIG()
 
     fb.save(outPath)
 
     _verifyOutput(outPath)
 
 
-def test_build_otf(tmpdir):
-    outPath = os.path.join(str(tmpdir), "test.otf")
+@pytest.mark.parametrize('out_filename, keep_glyph_names', [
+    # CFF 1.0 fonts should always use post table format 3.0
```

Comment:
So, what we're effectively testing here is whether the `keepGlyphNames` flag is ignored in the case of CFF 1.0, correct?

Reply:
That's right